src/CombFilter/FIRComb.cpp

```diff
@@ -8,3 +8,21 @@
 #include "FIRComb.h"
 
 
+Error_t CFIRComb::process (float **ppfInputBuffer, float **ppfOutputBuffer, int iNumberOfFrames)
```

Comment:
I see that you have made separate files for each filter. While I may be nitpicking when I say this, I think it would be better to have them in the same file as they are essentially the same function except for a difference in one line.